src/tezos/TezosConseilClient.ts

```diff
@@ -0,0 +1,127 @@
+import {ConseilOperator, ConseilQuery, ConseilSortDirection, ConseilServerInfo, ConseilQueryBuilder} from "../utils/v2/ConseilQuery";
+import {ConseilDataClient} from "../utils/v2/ConseilDataClient";
+
+/**
+ * Functions for querying the Conseil backend REST API v2
+ */
+export namespace TezosConseilClient {
```

Comment:
For the record, I am totally fine with this being a stateless class inheriting from a generic stateless class.

Reply:
all the classes were removed in this commit